ui/00_message/tutorial_guide/steps/tutorial_guide_step35.xml

```diff
@@ -22,5 +30,15 @@
 《港洞》　　…海洋任務に備える停船所
 
 《謁見の間》…白竜や重要人物との対話の場</original>
+    <translation>The White Dragon Temple is divided into several districts.
```

Comment:
Since the place is mentioned at the start of the description, using "temple" is fine afterwards. It's just unnecessary to capitalize it.